Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,175 @@
+//
+//  CDTFetchChanges.h
+//  CloudantSync
+//
+//  Created by Michael Rhodes on 31/03/2015.
+//  Copyright (c) 2015 IBM Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+@class CDTDatastore;
+@class CDTDocumentRevision;
+
+/**
+ CDTFetchChanges objects report changes to a database, that is, documents which have been
+ created, updated or deleted. This feature can be used to keep external datastores up to
+ date with changes in a database, for example to keep an external index of documents up
+ to date (we use CDTFetchChanges in the Query implementation, for example).
+
+ CDTFetchChanges provides callbacks for updated and created documents (documentChangedBlock)
+ and deleted documents (documentWithIDWasDeletedBlock). It also provides a completion block
+ for when all changes have been received and processed. The documentChangedBlock is passed the
```

Comment:
Since you mention the other blocks (which get a link in AppleDocs), perhaps add one here for the completed block. 

"It also provides a completion block for when all changes have been received and processed (fetchRecordChangesCompletionBlock)"